kondo/src/main.rs

```diff
@@ -49,6 +50,10 @@ struct Opt {
     #[arg(short, long, value_parser = parse_age_filter, default_value = "0d")]
     older: u64,
 
+    /// Generates completions for the specified shell
+    #[arg(short = 'c', long = "completions", value_enum)]
```

Comment:
```suggestion
    #[arg(long = "completions", value_enum)]
```

Completions is a relatively infrequent option so I'd like to reserve the shorthand `c` for something that is more common in future :)